### ISSUE TITLE
[BUG] Phantom rolled # (from 1 to 5)

### ISSUE THREAD

**Issue by MinerB4** [lukebemish/revamped_phantoms#14]
I was using version 1.1.2 on 1.12.1 Fabric 0.16.9 for a bit and noticed this the first time Phantoms spawned.

I believe this is because the wrong part of the `/random` command is being used.
`/random roll 5` will only broadcast to the executor while `/random value 5` will broadcast the message to all.
Side note: I don't know if you're using `/random` and I'm only assuming so.

**Comment by lukebemish** [comment]
...very confused about what you're talking about here. Please actually say what the error/issue is.

**Comment by MinerB4** [comment]
oops, would've helped

Yeah, it keeps printing in chat Phantom rolled **#** (from 1 to 5) in chat, with # being the random number.

**Comment by lukebemish** [comment]
Yes, and? What mods do you have installed? Have you confirmed that it's this mod that's causing it? Where's your log?
